Parsers/ASimAuthentication/Parsers/ASimAuthenticationCrowdStrikeFalconHost.yaml

```diff
@@ -0,0 +1,109 @@
+Parser:
+  Title: ASIM Authentication parser for CrowdStrike Falcon Endpoint Protection
+  Version: '0.1.0'
+  LastUpdated: Oct 26 2023
+Product:
+  Name: CrowdStrike Falcon Endpoint Protection
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: CrowdStrike Falcon Endpoint Protection Documentation
+  Link: 
+    https://falcon.us-2.crowdstrike.com/documentation/page/d88d9ed6/streaming-api-event-dictionary
+    https://falcon.us-2.crowdstrike.com/documentation/page/eb1587d1/siem-connector#mc98af8f
+Description: |
+  This ASIM parser supports normalizing CrowdStrike Falcon Endpoint Protection logs to the ASIM Authentication normalized schema. These events are captured through CrowdStrike Falcon Endpoint Protection data connector which allows you to easily connect your CrowdStrike Falcon Event Stream with Microsoft Sentinel.
+ParserName: ASimAuthenticationCrowdStrikeFalconHost
+EquivalentBuiltInParser: _ASim_Authentication_CrowdStrikeFalconHost
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+    [
+      "0", "Informational",
+      "1", "Informational",
+      "2", "Low",
+      "3", "Medium",
+      "4", "High",
+      "5", "High"
+  ];
+  let parser = (disabled: bool=false) {
+      CommonSecurityLog
+      | where not(disabled)
+      | where (DeviceVendor == "CrowdStrike" and DeviceProduct == "FalconHost")
+      | where DeviceEventCategory == "AuthActivityAuditEvent" and DeviceEventClassID in ("userAuthenticate", "twoFactorAuthenticate")
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
Please validate this condition in your live environment.
My doubt is, the current condition would never meet. As per my understanding 'DeviceEventClassID' field would contain all three values i.e. ("userAuthenticate", "twoFactorAuthenticate","AuthActivityAuditEvent")

Please correct me if I'm wrong.

Reply:
@vakohl 
No. In case of AuthActivityAuditEvent, DeviceEventClassID field would contain Operation value i.e.  "userAuthenticate", "twoFactorAuthenticate" and DeviceEventCategory field would contain EventType value. i.e. "AuthActivityAuditEvent"

![image](https://github.com/Azure/Azure-Sentinel/assets/107753570/203c4401-d93c-4a4c-9595-3623fdc0cc88)